README.md

```diff
@@ -1,8 +1,18 @@
 Go bindings for Roc
 -------------------
 
-*Work in progress!*
-
```

Comment:
Let's keep the "Work in progress" note for now. We still need to add more tests, documentation, CI, etc.

Reply:
done